gcpbuildcache/build.gradle.kts

```diff
@@ -16,7 +16,6 @@
  */
 
 plugins {
-    id("java-gradle-plugin")
```

Comment:
Why is this no longer needed?

Reply:
It's [applied automatically](https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html#plugin-publishing-plugin) by the `gradle-publish-plugin` now.